### PR TITLE
separate notification center/notification toast background color

### LINKED ISSUE #85588: Notification Toast Background
Currently the Notification Center background Colour and Notification Toast background Colour are the same.I would like the option to theme each one differently so as the notification Toast can stand out more and the Notification center can properties of the theme applied.

### PR DESCRIPTION
**Pull request by hypercubestart**
This PR fixes #85588 

Provides more customizable notification background coloring. This PR creates a new color called `notificationsToast.background` so notification toast can be customized separate from `notifications.background`. If `notificationsToast.background` is not set, the notification toast is use the color from `notifications.background`.

@Jabor047 would like further color customization such that onHover and onSelected background colors are changed as well, however, I couldn't think of an easy way of implementing this since it is implemented using CSS :hover :focus which cannot be changed using javascript. Please advise here.

----------------
Testing: Suggest to use extension called `Notification Tester`, edit settings.json for user and add properties like
```
{
	"workbench.colorCustomizations": {
		"notifications.background": ...,
                 "notificationsToast.background": ...
	}
}
```

### CODE REVIEW

File: src/vs/workbench/browser/parts/notifications/notificationsToasts.ts [hunk 1]
```diff
@@ -421,15 +421,19 @@ export class NotificationsToasts extends Themable {
...
 			t.toast.style.boxShadow = widgetShadowColor ? `0 0px 8px ${widgetShadowColor}` : '';
 
 			const borderColor = this.getColor(NOTIFICATIONS_TOAST_BORDER);
 			t.toast.style.border = borderColor ? `1px solid ${borderColor}` : '';
 		});
+
+		document.querySelectorAll('.notification-toast-container .monaco-list-rows')
+			.forEach((element: HTMLElement) => {
+				const backgroundColor = this.getColor(NOTIFICATIONS_TOAST_BACKGROUND);
+				element.style.background = backgroundColor ? backgroundColor : '';
```
Comment (ZuBB) [inline src/vs/workbench/browser/parts/notifications/notificationsToasts.ts]: ternary operator can be replaced with logical "OR". resulted statement will be shorter

**Review by bpasero — CHANGES_REQUESTED** [review]
I am not convinced just 1 new color is sufficient, given that you can change the background for toasts but pretty much nothing else, so I asked in https://github.com/microsoft/vscode/issues/85588#issuecomment-566613994

@hypercubestart the change is not clear to me. I would preserve as much of the `NOTIFICATIONS_BACKGROUND` usage as we have today, but just make the specific color key configurable from the outside (e.g. to take the one for toasts as needed).

File: src/vs/workbench/browser/parts/notifications/notificationsToasts.ts (hunk 1 above)
Comment (hypercubestart) [reply src/vs/workbench/browser/parts/notifications/notificationsToasts.ts]: thanks for the code review!
I wanted to keep the code style consistent, for example here,
https://github.com/microsoft/vscode/blob/fc1c307d093163d6355569a79bb6639e534d8934/src/vs/workbench/browser/parts/notifications/notificationsList.ts#L221-L232

Do you think shortening this statement will decrease readability at all?

**Review by fanyinghao — APPROVED** [review]

**Review by sowmyadharg — APPROVED** [review]

**Comment by hypercubestart** [comment]
@bpasero can you please re-review?

**Comment by bpasero** [comment]
The change is still not clear to me. Why the use of document.querySelectorAll which we never do for theme related things so far? Also, the discussion in https://github.com/microsoft/vscode/issues/85588#issuecomment-566613994 seems unresolved to me.

**Comment by bpasero** [comment]
It looks like this will not happen, so I will close this as stale soon unless I hear back.

**Pull request closed without merging**